pedalboard-web/src/components/PedalList.vue

```diff
@@ -0,0 +1,44 @@
+<script setup lang="ts">
+import { ref } from 'vue'
+import type { Prop, Ref } from 'vue'
+import { type Pedal } from '../models/pedal'
+import { type Knob } from '../models/knob'
+
+interface Props {
+    pedals: Pedal[]
+}
+
+const props = defineProps<Props>()
+
+</script>
+
+<template>
+  <h2>Pedals</h2>
+  <li v-for="pedal in pedals">
+    <div class="pedal">
+      <h3>{{ pedal.name }}</h3>
+      <li class="knobs" v-for="knob in pedal.knobs">
+        - {{ knob.name }}
+      </li>
+      <h3>{{ pedal.brand }}</h3>
+    </div>
```

Comment:
```suggestion
    <div class="pedal">
      <h3>{{ pedal.name }} - {{ pedal.brand }}</h3>
      <li class="knobs" v-for="knob in pedal.knobs">
        - {{ knob.name }}
      </li>
    </div>
```

I've noticed that the pedal name and brand are shown a bit too separated from each other. What do you think of this suggestion?

Reply:
I have made this *EXACT* same change in the next PR (edit-pedals). It looks way better like this. suggestion accepted!